java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -282,4 +282,37 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<ClusterValue<String[]>> time(Route route);
+
+    /**
+     * Returns <code>unix time</code> of the last DB save timestamp or startup timestamp if no save
```

Comment:
I notice the Redis documentation uses `UNIX TIME` (all caps) instead. Do we want to align it with the Redis docs?

Reply:
redis docs for [`time`](https://redis.io/docs/latest/commands/time/) command says "Unix timestamp"
How should we do? unix/epoch time/timestamp (+ the same in UC)

Reply:
Updated everywhere + for `time` command which also refers to unix time